Update the README files and also add the default value for the temporary directory parameter. (#278)
IntelLabs/vdms: user_defined_operations/README.md

## Detailed Instructions for new UDF
We now provide an example to add a new UDF `cardetect`. The `cardetect` operation detects cars in an image and creates a rectangle around all cars. This operation requires a pretrained model available in the form of `xml` file online.
1. Copy `user_defined_operations` directory to anywhere you want but on the same server that is running VDMS. Say you copy the folder in the `home` directory.

> 2. Copy `resources` directory (located at the root of the repository) next to the `user_defined_operations` directory. The folder structure you have now will look something like this;

Reviewer: ```suggestion
2. Copy `resources` directory (located at the root of the repository) into the `user_defined_operations` directory. The folder structure you have now will look something like this;
```

Author: Hi Chaunté, the "resources" and "user_defined_operations" directories should be siblings.
As they are siblings directories, could you please tell me if I should clarify that or if the use of "next" could confuse to the reader?
Thank you.

Author: Hi @cwlacewe and @rv355, as Rohit and I discussed by chat, I modified the README files to explain that the resources directory should be copied into the corresponding remote_function or user_defined_operations directory. Also, I modified the steps so the user can follow that approach.
Please let me know if you see any issues or if you have any suggestions.
Thank you

   |__udf_local.py

> |__resources

Reviewer 2: Shouldn't the `resources` directory be one level down inside `user_defined_operations`?

Author: Yeah, you're right. Thanks for noticing it.
